packages/data-store/src/migrations/sqlite/1708525189002-CreateUniformCredential.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class CreateUniformCredential1708525189002 implements MigrationInterface {
+  name = 'CreateUniformCredential1708525189002'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "UniformCredential" (
+                "id" varchar PRIMARY KEY NOT NULL,
+                "credential_type" varchar CHECK( "credential_type" IN ('vc', 'vp') ) NOT NULL,
+                "document_format" varchar CHECK( "document_format" IN ('JSON-LD', 'JWT', 'SD-JWT', 'MDOC') ) NOT NULL,
```

Comment:
Consistency. Either have the enums in caps or lowercase, but not one in upper and others in lower. In order to prevent issues with mapping DTO's either user string literals instead of enums, or prevent things like dashes and have the enum in the same case as the value.
Typeorm has enum support, but be aware that the DTOs cross from frontend to backend, so if you re-use the same enums between entities and DTO's you are now in trouble

Reply:
changed the enums to havesame name and value, and second made them all caps